### PR TITLE
DOCS-11992 First pass

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/mq-apis.adoc [hunk 1]
```diff
@@ -253,9 +253,13 @@ This returns output similar to:
 }
 ----
 
-=== Real-Time Statistics for Queues
+=== Current Statistics for Queues
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-apis.adoc:256]: I would leave the title as is. `Real-Time statistics for queues`

File: modules/ROOT/pages/mq-apis.adoc [hunk 2]
```diff
@@ -253,9 +253,13 @@ This returns output similar to:
...
 
-You can get an immediate snapshot of how many messages are in the queue and how many are in-flight (see xref::mq-understanding.adoc[Anypoint MQ Terminology] for what in-flight messages are). This data is updated in almost real-time. The Statistics API can fetch real-time statistics for multiple queues and returns a comma-separated list. 
+You can get a snapshot of the number of messages in the queue and how many are in-flight. 
+See xref::mq-understanding.adoc#glossary[Anypoint MQ Terminology] for a definition of in-flight messages.
+The data updates in almost real-time.
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-apis.adoc:260]: After almost real-time, i would add 

`The response from the Stats API  includes 60 seconds of inconsistency due
to internal post-processing activities`

This way we are elaborating what `almost real-time` means.

File: modules/ROOT/pages/mq-apis.adoc [hunk 3]
```diff
@@ -275,6 +279,15 @@ https://anypoint.mulesoft.com/mq/stats/api/v1/organizations/ORGANIZATION_ID/envi
 ]
 ----
 
+Use the Stats API only for statistics.
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-apis.adoc]: Post processing causing up to 60 seconds of inconsistency is also applicable to change is message statuses (to in-flight, from in-flight, and to deleted). For this reason, don't use the Stats API for syncing processing actions (e.g checking if the queue is empty etc.)

You can set a 60-second delay to account for the post-processing time

File: modules/ROOT/pages/mq-apis.adoc [hunk 4]
```diff
@@ -253,9 +253,17 @@ This returns output similar to:
...
-You can get an immediate snapshot of how many messages are in the queue and how many are in-flight (see xref::mq-understanding.adoc[Anypoint MQ Terminology] for what in-flight messages are). This data is updated in almost real-time. The Statistics API can fetch real-time statistics for multiple queues and returns a comma-separated list. 
+You can get a snapshot of the number of messages in the queue and how many are in-flight. 
+See xref::mq-understanding.adoc#glossary[Anypoint MQ Terminology] for a definition of in-flight messages.
+The data updates in almost real-time.
+
+Stats API responses and message status changes include up to 60 seconds of inconsistency due to internal post-processing activities.
+For this reason, don't use the Stats API for syncing processing actions,
+such as checking if the queue is empty.
+
+The Stats API can fetch statistics for multiple queues and returns a comma-separated list.
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-apis.adoc:267]: (from message to in-flight, from in-flight to message, and from in-flight to deleted)

**Review by kimayap03 — APPROVED** [review]
LGTM!

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**